db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+stache_1 = Stache.create(name: "Handlebar", description: "things", price: 5.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
+stache_2 = Stache.create(name: "Chevron", description: "things", price: 3.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
+stache_3 = Stache.create(name: "Dali", description: "things", price: 10.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
+stache_4 = Stache.create(name: "English", description: "things", price: 9.0, image_url: "http://images.clipartpanda.com/mustache-clipart-ecMA4abcn.svg")
```

Comment:
Useless assignment to variable - `stache_4`. Did you mean `stache_1`?<br>Line is too long. [152/80]